packages/core-sdk/resolvers/token-resolvers/resolvers/curve/src/query/__tests__/e2e/apiCalls.ts

```diff
@@ -0,0 +1,76 @@
+import { InvokeApiResult, Web3ApiClient } from "@web3api/client-js";
+
+import { Interface_TokenComponent } from "../../w3";
+
+export const isValidProtocolToken = async (
```

Comment:
We should be able to replace this with app codegen once we merge it so I don't see any need to preemptively refactor it just to replace it again.